hack/verify-examples-kind.sh

```diff
@@ -33,6 +33,9 @@ cleanup() {
   if [ "$CLEANED_UP" = "true" ]; then
     return
   fi
+
+  rm config/webhook/kustomization.yaml
```

Comment:
Why does this need to be here?

Reply:
Or if the `config/webhook/kustomization.yaml` is appropriate to be there, we can just leave it.